mvpa2/tests/test_statis.py

```diff
@@ -0,0 +1,104 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Unit tests for PyMVPA ..."""
+
+import unittest
+import numpy as np
+from itertools import combinations
+
+from mvpa2.algorithms.statis import Statis
+from mvpa2.misc.support import idhash
+from mvpa2.misc.data_generators import random_affine_transformation
+from mvpa2.base.dataset import hstack
+from mvpa2.testing.datasets import datasets
+from mvpa2.mappers.fx import mean_group_sample
+from mvpa2.testing import sweepargs
+from mvpa2.testing.tools import assert_almost_equal
+class StatisTests(unittest.TestCase):
+    """
+    Adapted test from Hyperalignment testing as they both aim to
+    align datasets into common space
+    """
+    @sweepargs(hstacked=(True, False))
+    def test_basic_alignment(self, hstacked):
+        statis = Statis(tables_attr='subject')
```

Comment:
How about adding another test, using data from Abdi et al 2012, Table 1, and verify that the output matches the one in the paper?